Wrappers/Python/test/test_Operator.py

```diff
@@ -997,3 +998,57 @@ def test_CompositionOperator_adjoint7(self):
                                                 2 * out2.as_array())
         numpy.testing.assert_array_almost_equal(d_out.as_array(),
                                                 2 * out2.as_array())
+
+
+    def test_CompositionOperator_norm1(self):
+
+        example_operator1 = Mock()
+        example_operator1.norm.return_value = 2
+
+        example_operator2 = Mock()
+        example_operator2.norm.return_value = 3
+
+        d = CompositionOperator(example_operator1, example_operator2)
+
+        out1 = 6
+        out2 = d.norm()
+
+        numpy.testing.assert_almost_equal(out2, out1)
```

Comment:
since you are using `int` as values you could use `assert_equal`

Reply:
Leaving this as assert_almost_equal as in the end we said to change values to float